test/connectivity/NetworkController/config/settings.json

```diff
@@ -0,0 +1,23 @@
+{
+  "StartAfter": "00:05:00",
+  "DockerUri": "unix:///var/run/docker.sock",
+  "NetworkId": "azure-iot-edge",
+  "NetworkControllerMode": "Offline",
+  "RunFrequencies": [
```

Comment:
do you know how to change these frequencies by environment variable if we want to change it during deployment?

Reply:
RunFrequencies__0__OfflineFrequency="00:00:10"

---
In reply to: [353566796](https://github.com/Azure/iotedge/pull/2040#discussion_r353566796) [](ancestors = 353566796)